WooCommerce/Classes/Yosemite/StoresManager.swift

```diff
@@ -103,6 +103,15 @@ class StoresManager {
         return self
     }
 
+    /// Prepares for changing the selected store and remains Authenticated.
+    ///
+    func resetAuthentication() {
```

Comment:
Just thinking out loud...I am wondering if this func name could be more precise? Maybe something like `removeDefaultStore`? Any thoughts here?